paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -284,6 +284,12 @@ internal class IntentConfirmationHandler(
     private fun confirmExternalPaymentMethod(
         paymentSelection: PaymentSelection.ExternalPaymentMethod
     ) {
+        /*
+         * In case of process death, we should store that we waiting for a payment result to return from a
+         * payment confirmation activity
+         */
+        storeIsAwaitingForPaymentResult()
```

Comment:
Since `PaymentSheet` no longer relies on the intent result as the source of truth for payment completion, we can now wait for the EPM result to return before continuing in `PaymentSheet`